src/Java.Interop.Tools.JavaSource/Java.Interop.Tools.JavaSource/SourceJavadocToXmldocGrammar.HtmlBnfTerms.cs

```diff
@@ -221,10 +231,9 @@ static IEnumerable<XElement> GetParagraphs (ParseTreeNodeList children)
 			public  readonly    NonTerminal PBlockDeclaration           = new NonTerminal (nameof (PBlockDeclaration), ConcatChildNodes);
 			public  readonly    NonTerminal PreBlockDeclaration         = new NonTerminal (nameof (PreBlockDeclaration), ConcatChildNodes);
 			public  readonly    NonTerminal InlineHyperLinkDeclaration  = new NonTerminal (nameof (InlineHyperLinkDeclaration), ConcatChildNodes);
-			public  readonly    NonTerminal IgnorableElementDeclaration = new NonTerminal (nameof (IgnorableElementDeclaration), ConcatChildNodes);
 			public  readonly    NonTerminal CodeElementDeclaration      = new NonTerminal (nameof (CodeElementDeclaration), ConcatChildNodes);
 
-			public  readonly    Terminal    InlineHyperLinkOpenTerm     = new RegexBasedTerminal ("<a href=", @"(?i)<a\s*href\s*=") {
+			public  readonly    Terminal    HtmlAElementStart           = new RegexBasedTerminal ("<a attr=", @"(?i)<a\s*.*=") {
```

Comment:
I'm not sure I understand why this rename is necessary, as it removes the "name similarity" from `InlineHyperLinkDeclaration`.

And the rationale for "inline hyperlink declaration" is from the [HTML spec](https://www.w3.org/TR/html4/struct/links.html#edef-A), wherein `<a/>` is an "inline" (non-block) element.

Reply:
The rename was to reflect that it will now captures`<a/>` elements with _any_ attribute, I've changed it back as it's not necessary.